docs/source/faq.rst

```diff
@@ -21,6 +21,21 @@ The IVI compliant VISA implementations available (`National Instruments NI-VISA`
 certain systems. We wanted to provide a compatible alternative.
 
 
+Are GBIP secondary addresses supported?
+---------------------------------------
+
+GPIB secondary addresses are supported in NI-VISA fashion, meaning that the
+secondary address is not 96 to 126 as transmitted on the bus, but 0 to 30.
+
+For expample, `GPIB0::9::1::INSTR` is the address of the first VXI module
+controlled by a GPIB VXI command module set to primary address `9`, while
+the command module itself is found at `GPIB0::9::0::INSTR`, which is distinct
+from a pure primary address like `GPIB0::9::INSTR`.
+
+``ResourceManager.list_resources()`` has become slower as a result,
+as it now needs to check 992 addresses per GPIB controller instead of just 31.
```

Comment:
Out of curiosity if there is no answer on a primary address is it possible to get a response with a secondary one ? If not it could help speed things up.

Reply:
Yes that is possible, the E1406A GPIB command module does not answer on its primary address, only on the primary address with a correct secondary address set, which allows to enumerate all modules in the VXI chassis.

The other two devices I tested do not care about the secondary address, they answer on the primary and on the primary + any secondary address.

It should be possible for a device to answer on the primary **and** have subdevices on secondary addresses, but I have never heard of one, maybe the ICS 4896 GPIB to quad serial converter, its product brief is not very clear on the topic.

Reply:
Could you add a comment detailing the possible behavior so that nobody gets the idea to try to optimize things later ?